service/lib/dinstaller/dbus/users.rb

```diff
@@ -0,0 +1,96 @@
+# Copyright (c) [2022] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "dbus"
+require "dinstaller/users"
+
+module DInstaller
+  module DBus
+    # YaST D-Bus object (/org/opensuse/YaST/Installer1)
+    #
+    # @see https://rubygems.org/gems/ruby-dbus
+    class Users < ::DBus::Object
+      PATH = "/org/opensuse/DInstaller/Users1".freeze
+      private_constant :PATH
+
+      USERS_INTERFACE = "org.opensuse.DInstaller.Users1".freeze
+      private_constant :USERS_INTERFACE
+
+      def initialize(logger)
+        @logger = logger
+        @root_password_set = false
```

Comment:
This looks like a leftover from a previous implementation. That attribute is now delegated to the backend, isn't?